projects/microhitl/examples/test_read_write.py

```diff
@@ -0,0 +1,11 @@
+from microhitl.microhitl import MicroHitl, Values, PinTypes
```

Comment:
Can you update this? What would be cool is a loop that blinks an arduino by doing HIGH, sleep, LOW, sleep, and then reads the value of a _different_ pin each loop.